bullet_train-super_scaffolding/lib/scaffolding/script.rb

```diff
@@ -53,6 +113,19 @@ def check_required_options_for_attributes(scaffolding_type, attributes, child, p
       {}
     end
 
+    data_type = if type == "image" && cloudinary_enabled?
+      "string"
```

Comment:
Oh, here's that special case I mentioned above. Maybe we remove `cloudinary_image` from that list? (Or do we want to leave it for backwards compatibility?)

Reply:
> Maybe we remove cloudinary_image from that list? (Or do we want to leave it for backwards compatibility?)

Right, I was planning on leaving it here for backwards compatibility. However, if we were to delete it, it technically wouldn't be breaking anything so I don't feel too strongly either way right now about leaving it or deleting it.